### ISSUE TITLE
Include `remake` in example for `ODEProblem

### ISSUE THREAD

**Issue by nonsciencemark** [SciML/SciMLBase.jl#488]

**Comment by ChrisRackauckas** [comment]
I think it's reasonable. You mean add it to the first tutorial?

**Comment by nonsciencemark** [comment]
Yes. I think that examples of `remake` like above would be useful in [in the first tutorial](https://docs.sciml.ai/ModelingToolkit/stable/tutorials/ode_modeling/#Your-very-first-ODE) as well as in [line 82 in the source](https://github.com/SciML/SciMLBase.jl/blob/master/src/problems/ode_problems.jl) so that it's visible with `?ODEProblem`[^but]. 

There are also cases in the [problem library](https://github.com/SciML/DiffEqProblemLibrary.jl/tree/master/lib/ODEProblemLibrary) which `?ODEProblem` leads to that create new `ODEProblems` instead of using `remake`.

Also [the part of the FAQ that mentions this functionality](https://docs.sciml.ai/ModelingToolkit/stable/basics/FAQ/#Using-ModelingToolkit-with-Optimization-/-Automatic-Differentiation) could be renamed so that it is more easily findable. Not sure what synonyms would be appropriate but searching e.g., "update/change/modify `parameters`/`state`/`tspan`" don't seem to get you to this information.

[^but]: maybe this would lead to bloat creep, but it does already have `plot`

**Comment by ChrisRackauckas** [comment]
100% agreed. 

> There are also cases in the [problem library](https://github.com/SciML/DiffEqProblemLibrary.jl/tree/master/lib/ODEProblemLibrary) which ?ODEProblem leads to that create new ODEProblems instead of using remake.

It would be good to get an issue open there. Those cases were probably just implemented before `remake` was created, so it was never updated.

> Also [the part of the FAQ that mentions this functionality](https://docs.sciml.ai/ModelingToolkit/stable/basics/FAQ/#Using-ModelingToolkit-with-Optimization-/-Automatic-Differentiation) could be renamed so that it is more easily findable. Not sure what synonyms would be appropriate but searching e.g., "update/change/modify parameters/state/tspan" don't seem to get you to this information.

"Updating or modifying states and parameters of an already built problem" ?

**Comment by nonsciencemark** [comment]
> "Updating or modifying states and parameters of an already built problem"

I think that would be better, yes.

> It would be good to get an issue open there

Will open a new issue over there when I have a bit of time
